### PR TITLE
Righto implementation

### PR DESCRIPTION
**Pull request by KoryNunn**

### CODE REVIEW

File: test [hunk 1]
```diff
@@ -54,6 +54,7 @@ test synchronous.js
 test callbacks.js
 test async.js
 test kgo.js
+test righto.js
```
Comment (davidchambers) [inline test]: Please update the readme as well. :)

**Comment by davidchambers** [comment]
There are a few stylistic changes I'd like you to make:

  - use two spaces for indentation;
  - use `if (…) {` and `} else {` (rather than `if(…){` and `}else{`); and
  - use `{key: value}` (rather than `{key:value}`).

Finally, please squash the commits. :)

**Comment by KoryNunn** [comment]
Done

**Comment by davidchambers** [comment]
Thanks. Two minor things: please use two spaces for indentation, and change `{ encoding: 'utf8' }` to `{encoding: 'utf8'}`. I'll merge the pull request once you've made these changes.

**Comment by KoryNunn** [comment]
also done

**Comment by davidchambers** [comment]
LGTM

**Pull request merged**